app/controllers/equipment_models_controller.rb

```diff
@@ -58,6 +58,17 @@ def show
     @max = @equipment_model.equipment_items.active.count
 
     @restricted = @equipment_model.model_restricted?(cart.reserver_id)
+
+    # For pending reservations table
```

Comment:
We're already querying for active reservations above (line 41), why don't we store `Reservation.active.for_eq_model(@equipment_model)` into a separate variable so we can just call off of that (e.g. `@today = @relevant_reservations.checkoutable` and `@upcoming = @relevant_reservations.reserved_in.date_range...`). That should help with the `MethodLength` check we failed and also keep things simpler.

Actually, as I'm thinking about it, it's probably easier to just include a single local variable `@pending`, and then in the view we can call @pending.checkoutable.count` to get the number of reservations that can be checked out today. In that case, we don't need the `future` scope (since we're including the checkoutable reservations in a single query).

Let me know if that makes sense; I'll leave a few other inline comments in the relevant places to flesh it all out.